src/gnatss/main.py

```diff
@@ -169,6 +182,28 @@ def get_transmit_times(
         right_on=constants.GPS_TIME,
     )
 
+    # Merge with rph data
+    if not rph_data.empty:
```

Comment:
base on your typehint, doing this if wouldn't work if `rph_data` is `None` since `NoneType` doesn't have the method `.empty`. It would be better suited to check `if isinstance(rph_data, pd.DataFrame) and not rph_data.empty`.

Reply:
Thank you for catching this oversight 😄  . Actually the `rph_data` cannot be `None` value, and the typehint is wrong. I will update my PR with this fix